packages/calypso-e2e/src/lib/pages/published/published-post-page.ts

```diff
@@ -123,7 +123,7 @@ export class PublishedPostPage {
 	 * @param {string} category Category to validate on page.
 	 */
 	async validateCategory( category: string ): Promise< void > {
-		await this.page.waitForSelector( `.cat-links :text("${ category }")` );
+		await this.page.waitForSelector( `a:text-is("${ category }")` );
```

Comment:
**question/non-blocking**: wouldn't this won't match against category text that also happens to be in the body?

eg. in the post body: 

"lorem ipsum sit dolor amet"

in the tag:

"lorem"

Reply:
It won't match a substring because we're using the exact match `:text-is()`, and we're also prefixing it with the anchor tag to be sure it targets only anchor elements, e.g.:

```
a:text-is("The Memoir")
```